### PR TITLE
Level one assets

### PR DESCRIPTION
**Pull request by Guysus**
added ground surface, rocks and debris, bushes and large cliff rocks.
The tree package will be able to use on all the levels

### CODE REVIEW

**Review by Saretzky — CHANGES_REQUESTED** [review]
there a a lot of assets, are they all necessary for level 1. and there's a conflict

File: Config/DefaultEngine.ini
Comment (Saretzky) [inline Config/DefaultEngine.ini:1]: why are you changing the default map to level 1?

**Review by Saretzky — CHANGES_REQUESTED** [review]
looking at this more there are definitely unnecessary assets in here. be sure you only inlude things we are using for level 1 specifically

**Review by Saretzky — CHANGES_REQUESTED** [review]
honestly, i would start from scratch, and only add in exactly what will be used for level 1 and nothing more. if the tree has a huge amount of assets, maybe find a more simple tree. 566 assets is too many

File: Content/LevelAssets/Trees_Assets_ForAllLevels/Meshes/SM_SK_Mannequin1.uasset
Comment (Saretzky) [inline Content/LevelAssets/Trees_Assets_ForAllLevels/Meshes/SM_SK_Mannequin1.uasset:1]: remove this

File: Season_trees_VOL_1.png
Comment (Saretzky) [inline Season_trees_VOL_1.png:1]: remove this

**Review by Saretzky — CHANGES_REQUESTED** [review]
Also, make sure everything is in the artwork folder

File: Season_trees_VOL_1.png
Comment (Saretzky) [inline Season_trees_VOL_1.png:1]: remove this

**Review by Saretzky — CHANGES_REQUESTED** [review]
look at the msPresets folder, move and rename, or delete what is unnecessary (Season_Trees_VOL1/Materials folder also). 

there is still a lot to rename in the Trees_Assets_ForAllLevels folder especially in the LOD, GLF, LF, and Luts folders

**Review by AMacLeod99 — APPROVED** [review]

File: Content/Season_trees_VOL1/Materials/M_ST_Inst.uasset
Comment (Saretzky) [inline Content/Season_trees_VOL1/Materials/M_ST_Inst.uasset:1]: rename and move this, or delete this

File: Season_trees_VOL_1.png
Comment (Saretzky) [inline Season_trees_VOL_1.png:1]: delete this

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_Lt_otm.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_Lt_otm.uasset:1]: Rename this

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_Nolf.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_Nolf.uasset:1]: Rename this

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_otm_c.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_otm_c.uasset:1]: Rename this

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_smr_c.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Luts/T_smr_c.uasset:1]: Rename this

**Review by Saretzky — CHANGES_REQUESTED** [review]
there are still many things that need to be renamed, I marked some, but most of the assets in the materials folder in the trees for all levels folder, and the assets in the foliage material folder

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_White.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_White.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Detail_Normal.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Detail_Normal.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Global_Wind.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Global_Wind.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Imposter.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Imposter.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_World_Normal.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_World_Normal.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Colourvar_ss.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Colourvar_ss.uasset:1]: Rename this to be more descriptive

File: Content/Season_trees_VOL1/Materials/M_ST_Inst.uasset
Comment (Saretzky) [inline Content/Season_trees_VOL1/Materials/M_ST_Inst.uasset:1]: delete, or rename and move this

File: Season_trees_VOL_1.png
Comment (Saretzky) [inline Season_trees_VOL_1.png:1]: delete this

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Default_Material.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Default_Material.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Detail_1.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Detail_1.uasset:1]: Rename this to be more descriptive

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Noise.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/T_Noise.uasset:1]: Rename this to be more descriptive

**Review by Saretzky — CHANGES_REQUESTED** [review]
change name for all files indicated and delete files indicated (since there are many review comments, you might need to expand it to see the full list). 

also, rename all the assets in the Trees_Assets_ForAllLevels/Textures/LODs folder

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Colourvar_ss.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Colourvar_ss.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_yl.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_yl.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_sscolor_1.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_sscolor_1.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_sscolor.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_sscolor.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_snow.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_snow.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_or.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_or.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_grn.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_grn.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_gr.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_gr.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_Normals.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_Glf_Normals.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_GL_RGB_Mask.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Leaf_Textures/T_GL_RGB_Mask.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Yell_brunch_leaf.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Grouped_Leaves_Textures/T_Yell_brunch_leaf.uasset:1]: change brunch to branch

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_smr_c.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_smr_c.uasset:1]: is this necessary?

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_otm_c.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_otm_c.uasset:1]: is this necessary?

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_otm.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_otm.uasset:1]: is this necessary?

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_Nolf.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Textures/Color_Bars_Textures/T_Color_Bar_Nolf.uasset:1]: is this necessary?

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_World_Normal.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_World_Normal.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Imposter.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Imposter.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Global_Wind.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Global_Wind.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Detail_Normal.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MaterialFunction/MF_Detail_Normal.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_White.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_White.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_Stripe.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/M_Stripe.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MI_Stripe_02.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MI_Stripe_02.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MI_Stripe_01.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/MI_Stripe_01.uasset:1]: change

File: Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/LODs/M_tl_yell_LOD.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/Trees_Assets_ForAllLevels/Materials/LODs/M_tl_yell_LOD.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/M_MS_Surface_Material.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/M_MS_Surface_Material.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Transmission.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Transmission.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Tiling.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Tiling.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_ObjAdjustments.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_ObjAdjustments.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_MapAdjustments.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_MapAdjustments.uasset:1]: change

File: Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Displacement.uasset
Comment (Saretzky) [inline Content/Artwork/LevelAssets/LevelOneAssets/Ground_Surfaces/Surface_Material/Functions/MF_Displacement.uasset:1]: change

**Review by Saretzky — CHANGES_REQUESTED** [review]

**Review by Jeff-Moreau — CHANGES_REQUESTED** [review]
This branch has conflicts with Hub.. needs to be fixed otherwise cant merge.
Also honestly there is too much in the branch.. break it up make it easier to follow

**Review by Saretzky — APPROVED** [review]
looks good

**Review by Jeff-Moreau — APPROVED** [review]
looks good

**Pull request merged**